code/game/turfs/simulated/walls_misc.dm

```diff
@@ -72,10 +72,9 @@
 	realappearance = new /obj/effect/clockwork/overlay/wall(src)
 	realappearance.linked = src
 
-/turf/simulated/wall/clockwork/Destroy()
+/turf/simulated/wall/clockwork/BeforeChange()
```

Comment:
whoops, this change is a stowaway

this makes the clockwork wall removal not leave the overlay behind